.spelling

```diff
@@ -41,7 +41,6 @@ api
 apis
 appimage
 applocker
-appveyor
```

Comment:
this might cause a break if AppVeyor is mentioned the the change log

Reply:
added back